src/Telemetry/class-telemetry-system.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Telemetry: Telemetry System abstract class
+ *
+ * @package Parsely\Telemetry
+ * @since   3.10.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Telemetry;
```

Comment:
An immediate concern I had was fatals between mu-plugins and wp-parsely because of this code re-use. However, looks like we have different namespaces (mu-plugins uses `Automattic\VIP\Parsely\Telemetry`) so that's good 😥

Reply:
Thanks for bringing this up. I'd also recommend not executing the mu-plugins telemetry code when the version of wp-parsely being loaded is 3.10.0 or greater.